orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/WaitForClusterDisableTask.groovy

```diff
@@ -68,33 +70,22 @@ class WaitForClusterDisableTask extends AbstractWaitForClusterWideClouddriverTas
     // null vs empty interestingHealthProviderNames do mean very different things to Spinnaker
     // a null value will result in Spinnaker waiting for discovery + platform, etc. whereas an empty will not wait for anything.
     if (interestingHealthProviderNames != null && interestingHealthProviderNames.isEmpty()) {
-      return false
+      return COMPLETED
     }
 
     if (!serverGroup.isPresent()) {
-      return false
+      return COMPLETED
     }
 
-    // Even if the server group is disabled, we want to make sure instances are down
-    // to prevent downstream stages (e.g. scaleDownCluster) from having to deal with disabled-but-instances-up server groups
+    // make sure that we wait for the disabled flag to be set
     def targetServerGroup = serverGroup.get()
-    if (targetServerGroup.isDisabled() || stage.context.desiredPercentage) {
```

Comment:
Is there something special here about the partial disables for RRB?

Reply:
not sure why there used to be, `waitForRequiredInstancesDownTask.hasSucceeded` knows how to deal with partial disables